target/linux/ramips/mt7621/config-6.6

```diff
@@ -44,6 +44,7 @@ CONFIG_CPU_SUPPORTS_MSA=y
 CONFIG_CRC16=y
 CONFIG_CRYPTO_DEFLATE=y
 CONFIG_CRYPTO_HASH_INFO=y
+CONFIG_CRYPTO_JITTERENTROPY=y
```

Comment:
Questionable.

Reply:
this seems to turn off this message in dmesg (line 3):
but apart from this I cannot see differences without and the router seems working, so I'm removing it

```
[   12.690342] kmodloader: loading kernel modules from /etc/modules.d/*
[   12.713106] mtk-eip93 1e004000.crypto: EIP93 Crypto Engine Initialized.
[   13.083889] jitterentropy: Initialization failed with host not compliant with requirements: 9
[   13.199692] Loading modules backported from Linux version v6.9.9-0-g28fdf4518483
[   13.207187] Backport generated by backports.git v6.1.97-1-29-gf1d24a3683b2
```